src/applications/personalization/dashboard/tests/e2e/claims-and-appeals.cypress.spec.js

```diff
@@ -52,8 +52,8 @@ describe('The My VA Dashboard Claims and Appeals section', () => {
     'when there are open claims or appeals but they have not been updated in the past 30 days',
```

Comment:
should the description of this test be "...in the past 60 days", not "30"?

Reply:
yep it should thanks

Reply:
oh that's the regular claims and appeals. V2 is 60 days. V1 is 30